supplier_app/__init__.py

```diff
@@ -32,6 +32,17 @@
         'Please click on on the following link to register' + '\n' +
         settings.COMPANY_INVITATION_URL
     },
+    'taxpayer_approval': {
+        'subject': 'Your taxpayer has been approved',
+        'body': 'You are ready to start using BriteSu' + '\n' +
+        'you can access the platform now and upload your invoices' +
+        settings.SUPPLIER_HOME_URL
+    },
```

Comment:
There is no need to concatenate this way ...BriteSu' + '\n' ... 
Also, can you use "{}".format(variable)?

Reply:
Yes!